Source/SG/SG.cs

```diff
@@ -52,13 +53,17 @@ public struct WaveformTimingConfiguration
             public double DutyCycle_Percent;
```

Comment:
This feels more like a generation timing configuration rather than attributes of the waveform

Reply:
I think that how GitHub was showing this it might have appeared like this was a property of `Waveform` but it's actually a part of `WaveformTimingConfiguration`. If you hit the expand button it will show you the entire code:
![image](https://user-images.githubusercontent.com/47825680/61390116-ccb15f00-a87f-11e9-96c1-0518173c118a.png)

Or check it out below. With this context, do you still think it is located in the incorrect place?

```c#
        public struct Waveform
        {
            public string WaveformName;
            public ComplexWaveform<ComplexSingle> WaveformData;
            public double SignalBandwidth_Hz;
            public double PAPR_dB;
            public double BurstLength_s;
            public double SampleRate;
            public int[] BurstStartLocations;
            public int[] BurstStopLocations;
        }
        public struct WaveformTimingConfiguration
        {
            public double DutyCycle_Percent;
```

Reply:
The way I look at it is "Does the waveform have a duty cycle?" or "Does the test require generation of the waveform conform to a specific duty cycle?" 
I guess the difference is, in WLANG SFP I could set the deadtime to make the waveform have a 50% duty cycle vs during generation i can dynamically configure the instrument to generate the waveform 50% of the time